app/views/jobseekers/profiles/job_preferences/_summary.html.slim

```diff
@@ -0,0 +1,58 @@
+ruby:
+  multistep = Jobseekers::JobPreferencesForm.from_record(profile.job_preferences)
```

Comment:
instantiating a class in a view also??

Reply:
@alexbowen - Yes, this view, as written, is very closely related to the particular implementation of the form and it is much healthier to instantiate it here rather than in a controller - especially since this is a partial which can be reused in any place without any need to modify the controller.

Once this is extracted as an abstract component, the form would be still instantiated in the partial to be passed to the component.